pkg/cli/root_test.go

```diff
@@ -27,8 +27,9 @@ import (
 func TestInitCfg(t *testing.T) {
 	cfgFile = filepath.Join(t.TempDir(), ".jvsctl.yaml")
 
-	if err := os.WriteFile(cfgFile, []byte(`server: https://example.com
-`), fs.ModePerm); err != nil {
+	if err := os.WriteFile(cfgFile, []byte(
+		`server: https://example.com
+jwks_endpoint: https://jvs.corp:8080/.well-known/jwks`), fs.ModePerm); err != nil {
```

Comment:
```suggestion
jwks_endpoint: https://example.com/.well-known/jwks`), fs.ModePerm); err != nil {
```

Reply:
This is probably my fault - I use `jvs.corp` in the godoc comment.